src/dlangui/graphics/glsupport.d

```diff
@@ -694,6 +694,8 @@ bool initGLSupport(bool legacy = false) {
     }
     if (!_glSupport) {
         Log.d("glSupport not initialized: trying to create");
+        int major = to!int(glGetString(GL_VERSION)[0 .. 1]);
```

Comment:
`glGetString(GL_VERSION)[0]` is a char and `to!int` simply casts it, e.g. if GL version is `2.1`, it will return `50`.